.ci/Makefile

```diff
@@ -0,0 +1,102 @@
+SHELL = /bin/bash -eo pipefail
+
+AWS_FOLDER = aws
+
+export AWS_FOLDER
+
+build:
+	rm -rf ../$(AWS_FOLDER) || true
+	mkdir -p ../$(AWS_FOLDER)
+	cd ../$(AWS_FOLDER); \
+		npm init -y; \
+		npm install --global-style https://github.com/elastic/apm-agent-nodejs#$(BRANCH_NAME); \
+		mkdir nodejs; \
+		mv node_modules nodejs
+env:
+	env
+dist: validate-branch-name build
+	rm -f $(BRANCH_NAME).zip || true
+	cd ../$(AWS_FOLDER) ; \
+		zip -r $(BRANCH_NAME).zip nodejs
+
+# List all the AWS regions
+get-all-aws-regions:
+	@aws \
+		ec2 \
+		describe-regions \
+		--region us-east-1 \
+		--output json \
+		--no-cli-pager \
+		| jq -r '.Regions[].RegionName' > .regions
+
+# Publish the given LAYER in all the AWS regions
+publish-in-all-aws-regions: validate-layer-name get-all-aws-regions
+	@mkdir -p $(AWS_FOLDER)
```

Comment:
A bit more of context:

- I chose to use `.ci` for storing the Makefile to avoid messing the existing one and decoupling what's related to the automation versus whats' related to the project itself.
- `make -C .ci <goal>` will use `.ci` as current working directory.
- npm install doesn't like to use `.ci` at any level so I was forced to use a different full path, for such, I used `../${AWS_FOLDER}`. Though, it was not as elegant as I wanted but I didn't see the need to complicate this a bit more.
- 

Let me consolidate the AWS location to `../${AWS_FOLDER}`

Reply:
> make -C .ci <goal> will use .ci as current working directory.

Ah, right. I forgot my `make` usage.

> Let me consolidate the AWS location to `../${AWS_FOLDER}`

Or perhaps to `../build/aws`, because we already have the "build/..." in .gitignore and eslint ignorePatterns.

Reply:
https://github.com/elastic/apm-agent-nodejs/pull/2585/commits/887962b830f27cda2384e6ce0fd25975a2bf6384 is the one with `../build/aws`, let me test it locally and I'll ping you for another review